daemon/mgr/container.go

```
@@ -1299,6 +1302,29 @@ func (mgr *ContainerManager) Resize(ctx context.Context, name string, opts types
 	return mgr.Client.ResizeContainer(ctx, c.ID, opts)
 }
 
+// Wait stops processing until the given container is stopped.
+func (mgr *ContainerManager) Wait(ctx context.Context, name string) (types.ContainerWaitOKBody, error) {
+	c, err := mgr.container(name)
+	if err != nil {
+		return types.ContainerWaitOKBody{
```

please return the error and empty `types.ContainerWaitOKBody` directly.